### DISCUSSION TITLE
Share command ideas.

### DISCUSSION THREAD

**Discussion by tupoy-ya** [YimMenu/YimMenu#758] [Ideas]
If you have command ideas that you think will be useful in YimMenu you can post them here.

Template:
```markdown
- `command_name`
 Command description.

- `other_command`
 Other description. Example: `other_command arg1`.
```

**Comment by tupoy-ya** [comment]
- `help`
 Get command description if no command is specified show generic help about commands.

- `clear`
 Clears the console window. Only useful after console is implemented.

- `alias`
 Lets you create custom commands from other commands.

- `act`
 Lets you play one of predefined animations. Example: `act dance`.

- `giveweapon`
 Gives player a weapon from name. Example: `giveweapon playername weapon_flaregun`.

- `cleartasks`
 Clears tasks from players ped.

- `stoploading`
Debug command that stops loading screen.
Possible implementation: 
```cpp
if (CAM::IS_SCREEN_FADED_OUT())
    CAM::DO_SCREEN_FADE_IN(0);
SCRIPT::SHUTDOWN_LOADING_SCREEN();
```

**Reply by alyxme** [reply]
Instead of `giveweapon playername weapon_flaregun` Id remove `playername` entering due struggle to input certain nicknames and use "Set selected" ID from context menu. That could be useful also for ease commands input for entities/cars/peds too.
